src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -87,6 +90,16 @@ public Routes(
     }
 
     /**
+     * Creates a {@link Flow} based on executing the routes asynchronously.
+     *
+     * @return a new {@link Flow}
+     */
+    public Flow<HttpRequest, HttpResponse, NotUsed> flow() {
+        return Flow.<HttpRequest>create()
```

Comment:
Is there something specific you foresee encapsulating in this method that would be internal to the routes definition? Otherwise, I'm not seeing the point of this method in this class (as opposed to for example outside this class encapsulating an instance in the flow).